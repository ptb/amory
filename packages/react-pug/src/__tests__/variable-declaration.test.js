import testHelper from './test-helper';

testHelper(__dirname + '/variable-declaration.input.js');
