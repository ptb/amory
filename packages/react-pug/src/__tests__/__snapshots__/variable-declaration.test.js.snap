// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JavaScript output: transformed source code 1`] = `
"let _x;

module.exports = <div>{(_x = 1, undefined)}{(_x = _x + 40, undefined)}{(_x++, undefined)}<div>{_x}</div></div>;"
`;

exports[`html output: generated html 1`] = `
<div>
  <div>
    42
  </div>
</div>
`;

exports[`static html output: static html 1`] = `"<div><div>42</div></div>"`;
